tests/test_mkfunding.py

```diff
@@ -24,20 +24,22 @@
 LOCAL_FUNDING_PRIVKEY = '0000000000000000000000000000000000000000000000000000000000000010'
 REMOTE_FUNDING_PRIVKEY = '0000000000000000000000000000000000000000000000000000000000000020'
 
+
 def subprocess_run(args):
     try:
-        response = subprocess.run(args,
-                                  timeout=TIMEOUT,
-                                  stderr=subprocess.PIPE,
-                                  stdout=subprocess.PIPE)
+        response = subprocess.run(
```

Comment:
nit: squash formatting changes into the original commit that made them (make it look as though you never had formatting issues in the commit history)

Reply:
Okay, I ran "git rebase -i HEAD~5" and squashed all but the first commit, and then committed and ran "git push --force origin blackbox-mkfunding". Now I see 1 commit in GitHub instead of 5. Thanks for the suggestion!

Reply:
Is there anything more I need to do on the PR?